src/main/java/ch/njol/skript/util/Utils.java

```diff
@@ -841,4 +841,34 @@ public int hashCode() {
 
 	}
 
+	/**
+	 * Converts an alphanumeric string into a valid UUID string
+	 * @param input alphanumeric string
+	 * @return Valid UUID string
+	 */
+	@Nullable
+	public static String convertUUID(String input) {
+		if (input.isEmpty()) {
+			return "00000000-00000000-00000000-00000000";
+		}
+		String[] parts = input.replaceAll("[^a-zA-Z0-9-]", "").split("-");
+		StringBuilder cleanInput = new StringBuilder();
+		for (String part : parts) {
+			cleanInput.append(part);
+		}
+		String result = cleanInput.toString().substring(0, Math.min(32, cleanInput.length()));
+		result = String.format("%32s", result).replace(' ', '0');
+
+		StringBuilder uuid = new StringBuilder();
+		uuid.append(result.substring(0, 8))
+			.append("-")
+			.append(result.substring(8, 16))
+			.append("-")
+			.append(result.substring(16, 24))
+			.append("-")
+			.append(result.substring(24, 32));
+
+		return uuid.toString();
+	}
```

Comment:
We should not allow malformed UUIDs to become valid UUIDs

Reply:
like for example this means "Never gonna give you up, never gonna let you 32 uio4vmu93mnv 35cn 3o4inc5h43t l"  is a valid UUID.

Reply:
I mean, technically it wouldnt tell the user it was an error, but the final value would be
"Nevergon-nagiveyo-uupnever-gonnalet"

Reply:
yes, that's bad! it's unintuitive behavior and should not do anything if the input isn't valid.
Plus, that's not even a valid uuid as output. UUIDs are hexadecimal as strings.